README.md

```diff
@@ -170,17 +170,16 @@ The following instructions all pertain to `provider/resources.go`, in the sectio
 
 **Note:** If you make revisions to code in `resources.go`, you must re-run the `make tfgen` target to regenerate the schema.  Pulumi providers use Go 1.16, which does not have the ability to directly embed text files.  The `make tfgen` target will take the file `schema.json` and serialize it to a byte array so that it can be included in the build output.  (Go 1.17 will remove the need for this step.)
 
-## Sample Program and End-to-end Testing
+## Sample Program
 
 In this section, we will create a Pulumi program in TypeScript that utilizes the provider we created to ensure everything is working properly.
 
 1. Create an account with the provider's service and generate any necessary credentials, e.g. API keys:
     * Email: bot@pulumi.com
     * Password: (Create a random password in 1Password with the  maximum length and complexity allowed by the provider.)
     * Ensure all secrets (passwords, generated API keys) are stored in Pulumi's 1Password vault.
-    * Enter any secrets consumed by integration tests as repository-level secrets in GitHub.  These will be used by the integration tests during the CI/CD process.
-1. Generate GitHub workflows per [the instructions in the ci-cmgmt repository](https://github.com/pulumi/ci-mgmt/#readme) and copy to `.github/` in this repository.
-1. Copy the binary generated by the build and place it in your `$PATH` (`$GOPATH/bin` is a convenient choice), e.g.:
+
+1. Copy the `pulumi-resource-*` binary generated by `make_build_sdks` and place it in your `$PATH` (`$GOPATH/bin` is a convenient choice), e.g.:
```

Comment:
A couple of things here:

1. `make provider` is the target that creates `pulumi-resource-foo` and `pulumi-tfgen-foo`.
1. The wilcard would need to be `pulumi-*-foo`.

Reply:
Hm, I was thinking that we don't need the `tfgen` binary in the $PATH for the `pulumi up` example. But I take your point about using `foo`.

Reply:
Ok, then I would expect it to read "Copy the `pulumi-resource-foo` binary..."

Reply:
yep 👍 done